### PR TITLE
feat: ja translate develop/api-reference/index

### LINKED ISSUE #191: [JP translaiton] API reference
https://github.com/momentohq/public-dev-docs/blob/main/i18n/ja/docusaurus-plugin-content-docs/current/develop/api-reference/index.mdx

### PR DESCRIPTION
**Pull request by riywo**
Close #191

### CODE REVIEW

**Comment by riywo** [comment]
@poppoerika Check this out!

File: i18n/ja/docusaurus-plugin-content-docs/current/develop/api-reference/index.mdx [hunk 1]
```diff
@@ -12,257 +12,252 @@ import { SdkExampleSnippets } from "@site/src/components/SdkExampleSnippets";
...
 | --------- | ------ | ------------------------------- |
-| nextToken | String | Token for pagination of caches. |
+| nextToken | String | キャッシュ一覧のページネーショントークン |
 
 <SdkExampleSnippets snippetId={'API_ListCaches'} />
 
 ### Flush cache
 
-Flushes all data from a cache
+キャッシュ内の全てのデータを真っ新にします。
```
Comment (poppoerika) [inline i18n/ja/docusaurus-plugin-content-docs/current/develop/api-reference/index.mdx]: Can we just say `フラッシュ`?

File: i18n/ja/docusaurus-plugin-content-docs/current/develop/api-reference/index.mdx [hunk 2]
```diff
@@ -12,257 +12,252 @@ import { SdkExampleSnippets } from "@site/src/components/SdkExampleSnippets";
...
 
-Attributes:
-
-| Name      | Type   | Description                      |
+| 名前      | 型   | 説明                      |
 | --------- | ------ | -------------------------------- |
-| cacheName | String | Name of the cache to be flushed. |
+| cacheName | String | 真っ新にするキャッシュの名前 |
```
Comment (poppoerika) [inline i18n/ja/docusaurus-plugin-content-docs/current/develop/api-reference/index.mdx]: same comment as https://github.com/momentohq/public-dev-docs/pull/196/files#r1186346023

File: i18n/ja/docusaurus-plugin-content-docs/current/develop/api-reference/index.mdx [hunk 3]
```diff
@@ -12,257 +12,252 @@ import { SdkExampleSnippets } from "@site/src/components/SdkExampleSnippets";
 // plugin will transform instances of SdkExampleSnippets to SdkExampleSnippetTabs
 import { SdkExampleSnippetTabs } from "@site/src/components/SdkExampleSnippetTabs";
 
-# API reference for Momento Cache
+# Momento Cache の API リファレンス
 
 ## Control APIs
```
Comment (poppoerika) [inline i18n/ja/docusaurus-plugin-content-docs/current/develop/api-reference/index.mdx]: `コントロールAPIs`

File: i18n/ja/docusaurus-plugin-content-docs/current/develop/api-reference/index.mdx [hunk 4]
```diff
@@ -12,257 +12,252 @@ import { SdkExampleSnippets } from "@site/src/components/SdkExampleSnippets";
...
 <SdkExampleSnippets snippetId={'API_FlushCache'} />
 
 :::tip
 
-While you could use [Delete Cache](#delete-cache), then [Create Cache](#create-cache) to mimic this, the FlushCache API keeps the settings and only deletes the data in the cache.
+これは、[Delete Cache](#delete-cache) してから [Create Cache](#create-cache) することで似た様なことができますが、 FlushCache API は設定を保持したままキャッシュ内のデータだけを削除してくれます。
 
 :::
 
 ## Data APIs
```
Comment (poppoerika) [inline i18n/ja/docusaurus-plugin-content-docs/current/develop/api-reference/index.mdx]: `データAPIs`

File: i18n/ja/docusaurus-plugin-content-docs/current/develop/api-reference/index.mdx [hunk 5]
```diff
@@ -12,257 +12,252 @@ import { SdkExampleSnippets } from "@site/src/components/SdkExampleSnippets";
...
     * Stored
     * NotStored
     * Error
 
-    See [response objects](./response-objects.md) for specific information.
+詳細な情報は、[レスポンスオブジェクト](./response-objects.md) をご覧下さい。
 
 </details>
 
 ## Time to Live APIs
```
Comment (poppoerika) [inline i18n/ja/docusaurus-plugin-content-docs/current/develop/api-reference/index.mdx]: `生存期間(Time to Live)APIs`

File: i18n/ja/docusaurus-plugin-content-docs/current/develop/api-reference/index.mdx [hunk 6]
```diff
@@ -12,257 +12,252 @@ import { SdkExampleSnippets } from "@site/src/components/SdkExampleSnippets";
...
+| 名前      | 型   | 説明                                     |
 | --------- | ------ | ----------------------------------------------- |
-| cacheName | String | Name of the cache.                              |
-| key       | String \| bytes | The key under which the value's TTL is to be decreased. |
-| ttl       | Duration | Time to live that you want to decrease to. |
+| cacheName | String | キャッシュの名前                              |
+| key       | String \| bytes | 値のTTL が減少されるキー |
+| ttl       | Duration | キャッシュ内の生存期間に設定したい秒数 |
 
 ### Collection data types (CDTs)
```
Comment (poppoerika) [inline i18n/ja/docusaurus-plugin-content-docs/current/develop/api-reference/index.mdx]: `コレクションデータ型(CDTs)`

File: i18n/ja/docusaurus-plugin-content-docs/current/develop/api-reference/index.mdx [hunk 7]
```diff
@@ -12,257 +12,252 @@ import { SdkExampleSnippets } from "@site/src/components/SdkExampleSnippets";
...
 
-Collections may contain different types of structures depending on your use case. Supported data types are:
+集合型は、ユースケースに依っては異なる構造の型を含む場合もあります。サポートしているデータ型は以下の通りです:
```
Comment (poppoerika) [inline i18n/ja/docusaurus-plugin-content-docs/current/develop/api-reference/index.mdx]: `集合型` -> `コレクション型`

File: i18n/ja/docusaurus-plugin-content-docs/current/develop/api-reference/index.mdx [hunk 8]
```diff
@@ -12,257 +12,252 @@ import { SdkExampleSnippets } from "@site/src/components/SdkExampleSnippets";
...
+* [辞書型](./dictionary-collections.md)は、順序のない field:value のペアを保存するのに使用されます。
+* [リスト型](./list-collections.md)は、順序のある要素の集合で、各要素が挿入された順番にソートされています。
+* [セット型](./set-collections.md)は、順序のない、ユニークな文字列形式の要素の集合です。
+* [ソート済セット型](./sorted-set-collections.md)は、ユニークな要素の順序がある集合です。各要素は value:score のペアで表現されます。
 
-* [Dictionaries](./dictionary-collections.md) are used to store unordered field:value pairs.
-* [Lists](./list-collections.md) are a collection of ordered elements, sorted in the sequence each element was inserted.
-* [Sets](./set-collections.md) are an unordered collection of unique elements in string format.
-* [Sorted Sets](./sorted-set-collections.md) are an ordered collection of unique elements.  Each element contains a value:score pair.
+より詳しい利用方法の情報は、[集合データ型](./../datatypes.md)をご覧下さい。
```
Comment (poppoerika) [inline i18n/ja/docusaurus-plugin-content-docs/current/develop/api-reference/index.mdx]: https://github.com/momentohq/public-dev-docs/pull/196/files#r1186357403

**Comment by riywo** [comment]
@poppoerika Addressed your comments! https://github.com/momentohq/public-dev-docs/compare/4a9792214d103319cda608d49b90cb63b484829c..6aa33c7236cf1aa9c7bcce882be4fda27ad46736

**Review by poppoerika — APPROVED** [review]

**Pull request merged**